package rtidb.common;

message KvPair {
    optional int64 time = 1;
    optional bytes value = 2;
}

message KvList {
    optional string pk = 1;
    repeated KvPair pairs = 2;
}

message ColumnDesc {
    required string name = 1;
    optional string type = 2;
    optional bool add_ts_idx = 3; 
    optional bool is_ts_col = 4;
    optional uint64 ttl = 5;
}

message ColumnKey {
    repeated string key_name = 1;
    optional string ts_name = 2;
}
